src/views/partials/nav.ejs

```diff
@@ -24,7 +24,7 @@
 
 			<ul class="navbar-nav">
 				<li class="nav-item logged-in">
-					<a class="nav-link <% if (page === 'groups') { %> active <% } %>" id="groups-nav" href="/groups">Groups</a>
+					<a class="nav-link <% if (page === 'groups') { %> active <% } %>" id="groups-nav" href="/bills">Bills</a>
```

Comment:
if (page === 'bills')

Reply:
Oh oops, had it locally, didn't finish pushing it